NeuroID/src/main/java/com/neuroid/tracker/service/NIDJobServiceManager.kt

```diff
@@ -16,14 +18,14 @@ import java.util.concurrent.TimeUnit
 
 object NIDJobServiceManager {
 
-    private var jobCaptureEvents: Job? = null
+    internal var jobCaptureEvents: Job? = null
     var isSendEventsNowEnabled = true
 
     @Volatile
     var userActive = true
-    private var clientKey = ""
+    internal var clientKey = ""
     private var application: Application? = null
-    private var endpoint: String = ""
+    internal var endpoint: String = ""
```

Comment:
Are we still supporting configuring endpoints @kevinsites ?

Reply:
Naw we can probably remove it, I am pretty sure we have removed all the references to it anyways

Reply:
I think we should keep this in there. We do have a development endpoint as well (https://receiver.neuro-dev.com) that we support.